rest/route/project.go

```diff
@@ -369,12 +369,6 @@ func (h *projectIDPatchHandler) Run(ctx context.Context) gimlet.Responder {
 					Message:    "cannot enable git tag versions without a version definition",
 				})
 			}
-			if len(mergedProjectRef.GitTagAuthorizedUsers) == 0 && len(mergedProjectRef.GitTagAuthorizedTeams) == 0 {
```

Comment:
Can you please explain a big more about why this is no longer necessary? Do we not need to still check to make sure that there are authorized users or teams? 
Is the theory that this validation now happens somewhere else, like in the middleware? Where does it now happen?

Reply:
We're just not going to validate it anymore. This was more of a courtesy validation but it's now more inline with how other permissions work. If we still wanted to validate that there is someone authorized we could also do a DB query to check if there are any users with the permission in Mana, but I think that's overkill. What if I wanted to first define the git tag tasks and regexes, and _then_ request permission in Mana?

Reply:
We do have validation elsewhere that the user who pushed the tag is part of one of these lists or has been granted the permission from Mana, which is the important part.